layouts/custompage/release.html

```diff
@@ -32,17 +32,42 @@ <h1>Download</h1>
     </tr>
   </thead>
   {{range first 5 (where (where .Site.Pages "Section" "release") ".Params.linked" true) }}
+
+    <!-- Define the suffix of the checksum file for each release -->
+    {{ $checksum_suffix := "mds" }}
+    <!-- if the version >= 3.3 -->
+    {{ if .File.BaseFileName | replaceRE "^([0-9]+.[0-9]+).*" "$1" | float | le 3.3 }}
+      {{ $checksum_suffix = "sha512" }}
+    <!-- else if the version > 3.2.0, 3.1.2, 2.9.2, 2.8.5 -->
+    {{ else if .File.BaseFileName | replaceRE "^([0-9]+.[0-9]).*" "$1" | eq "3.2" }}
+      {{ if .File.BaseFileName | replaceRE "^[0-9]+.[0-9]+.([0-9]+).*" "$1" | float | lt 0 }}
```

Comment:
In branch-3.2, 3.2.0  is with mds and 3.2.1 is sha512. So it should be **lt 1**?

Similarly, for branch-3.1, 3.1.0, 3.1.1 and 3.1.2 is mds 3.1.3 is sha512. So it should be **lt 3** right?

branch-2.9, should it be **lt 3**?
branch-2.8, should it be **lt 6**?

Reply:
`lt` means lower than (`<`), so this is okay. `le` is used for lower than or equal to (`<=`).